tests/conftest.py

```diff
@@ -35,3 +35,7 @@ def pytest_addoption(parser: pytest.Parser):
     parser.addoption("--test-db-from-config", action="store_true", default=False)
     # Number of WordPress units should the test deploy
     parser.addoption("--num-units", action="store", type=int, default=1)
+    # A directory to store screenshots generated by test_upgrade
+    parser.addoption("--screenshot-dir", action="store", default="")
+    # WordPress docker image built for the WordPress charm, default: localhost:32000/wordpress:test
```

Comment:
Is the default value needed in the comment? The code below seems to be quite readable in terms of the default value

Reply:
Removed, thanks!